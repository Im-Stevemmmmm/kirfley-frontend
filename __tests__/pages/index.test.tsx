import { MockedProvider } from '@apollo/client/testing';
import ReactDOM from 'react-dom';
import Homepage from '../../pages/index';

it('renders correctly', () => {
  const container = document.createElement('div');
  ReactDOM.render(
    <MockedProvider>
      <Homepage />
    </MockedProvider>,
    container
  );

  expect(container).toMatchSnapshot();
});
