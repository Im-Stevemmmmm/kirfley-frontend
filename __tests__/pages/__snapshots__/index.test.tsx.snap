// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div>
    <div>
      <div
        id="logo"
      >
        <div
          id="fernImg"
        >
          <div
            style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
          >
            <div
              style="box-sizing: border-box; display: block; max-width: 100%;"
            >
              <img
                alt=""
                aria-hidden="true"
                role="presentation"
                src="data:image/svg+xml;charset=utf-8,<svg width=\\"50\\" height=\\"50\\" xmlns=\\"http://www.w3.org/2000/svg\\" version=\\"1.1\\"/>"
                style="max-width: 100%; display: block;"
              />
            </div>
            <img
              decoding="async"
              src="/_next/image?url=%2Ffern.svg&w=320&q=75"
              srcset="/_next/image?url=%2Ffern.svg&w=320&q=75 1x"
              style="visibility: visible; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
            />
          </div>
        </div>
        <h2>
          Fern
        </h2>
      </div>
      <div
        id="links"
      >
        <button
          class="standard"
        >
          Log In
        </button>
        <button
          class="inverted"
        >
          Sign Up
        </button>
      </div>
    </div>
    <div
      id="banner"
    >
      <title>
        Fern
      </title>
      <h1>
        Fern
      </h1>
      <div
        id="horizontalBarContainer"
      >
        <div
          class="fernImg mirrorHorizontal"
          style="right: -200px;"
        >
          <div
            style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
          >
            <div
              style="box-sizing: border-box; display: block; max-width: 100%;"
            >
              <img
                alt=""
                aria-hidden="true"
                role="presentation"
                src="data:image/svg+xml;charset=utf-8,<svg width=\\"100\\" height=\\"100\\" xmlns=\\"http://www.w3.org/2000/svg\\" version=\\"1.1\\"/>"
                style="max-width: 100%; display: block;"
              />
            </div>
            <img
              decoding="async"
              src="/_next/image?url=%2Ffern.svg&w=320&q=75"
              srcset="/_next/image?url=%2Ffern.svg&w=320&q=75 1x"
              style="visibility: visible; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
            />
          </div>
        </div>
        <div
          id="horizontalBar"
        />
        <div
          class="fernImg"
          style="left: -200px;"
        >
          <div
            style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
          >
            <div
              style="box-sizing: border-box; display: block; max-width: 100%;"
            >
              <img
                alt=""
                aria-hidden="true"
                role="presentation"
                src="data:image/svg+xml;charset=utf-8,<svg width=\\"100\\" height=\\"100\\" xmlns=\\"http://www.w3.org/2000/svg\\" version=\\"1.1\\"/>"
                style="max-width: 100%; display: block;"
              />
            </div>
            <img
              decoding="async"
              src="/_next/image?url=%2Ffern.svg&w=320&q=75"
              srcset="/_next/image?url=%2Ffern.svg&w=320&q=75 1x"
              style="visibility: visible; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
            />
          </div>
        </div>
      </div>
      <h2
        class="wrappedText"
      >
        The Unbiased Social Media Platform
      </h2>
      <div
        id="authButtons"
      />
    </div>
    <div
      id="sectionOne"
    >
      <div
        id="rightColumn"
      >
        <div
          id="authCard"
        >
          <h1>
            Log In or Sign Up
          </h1>
        </div>
      </div>
      <div
        id="leftColumnContainer"
      >
        <div
          id="leftColumn"
        >
          <h1>
            The Unbiased Social Media Platform
          </h1>
          <p>
            Enjoy a free application where you can communicate with others without fear of your ideas being censored. Or some other random description like this. This is a placeholder!
          </p>
          <button
            class="inline"
          >
            Expirement
            <span>
               
              <div
                style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
              >
                <div
                  style="box-sizing: border-box; display: block; max-width: 100%;"
                >
                  <img
                    alt=""
                    aria-hidden="true"
                    role="presentation"
                    src="data:image/svg+xml;charset=utf-8,<svg width=\\"10\\" height=\\"10\\" xmlns=\\"http://www.w3.org/2000/svg\\" version=\\"1.1\\"/>"
                    style="max-width: 100%; display: block;"
                  />
                </div>
                <img
                  decoding="async"
                  src="/_next/image?url=%2Farrow.svg&w=320&q=75"
                  srcset="/_next/image?url=%2Farrow.svg&w=320&q=75 1x"
                  style="visibility: visible; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                />
              </div>
            </span>
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
